pkg/aks/create_test.go

```diff
@@ -56,3 +58,381 @@ var _ = Describe("CreateResourceGroup", func() {
 		Expect(err.Error()).To(ContainSubstring("failed to create resource group"))
 	})
 })
+
+var _ = Describe("newManagedCluster", func() {
+	var (
+		mockCtrl             *gomock.Controller
+		workplacesClientMock *mock_services.MockWorkplacesClientInterface
+		clusterSpec          *aksv1.AKSClusterConfigSpec
+		cred                 *Credentials
+	)
+
+	BeforeEach(func() {
+		mockCtrl = gomock.NewController(GinkgoT())
+		workplacesClientMock = mock_services.NewMockWorkplacesClientInterface(mockCtrl)
+		clusterSpec = &aksv1.AKSClusterConfigSpec{
```

Comment:
Can you see if you can define this as json in an embedded json test file and then `json.Unmarshal` it into an `aksv1.AKSClusterConfigSpec` object?

Reply:
I'd like to keep it as is since we don't reuse this cluster spec in other tests. It should be easier to see what values are set this way.